File: app/src/main/java/com/stone/framework/ui/splash/SplashContact.java
package com.stone.framework.ui.splash;

import com.stone.framework.frame.mvp.BasePresenter;
import com.stone.framework.frame.mvp.BaseView;

/**
 * Created by cdy on 2017/12/13.
 */

public interface SplashContact {

    interface View extends BaseView {
//        void showAdvImg(SplashEntity entity);

        void onError();
    }
    abstract class Presenter extends BasePresenter<View> {
        abstract void loadAdvData();
    }
}

File: app/src/main/java/com/stone/framework/ui/activity/splash/SplashContact.kt
package com.stone.framework.ui.activity.splash


import com.stone.lib.common.mvp.BasePresenter
import com.stone.lib.common.mvp.BaseView

interface SplashContact {

    interface View : BaseView {
        //        void showAdvImg(SplashEntity entity);

        fun onError()
    }

    abstract class Presenter : BasePresenter<View>() {
        internal abstract fun loadAdvData()
    }
}
